desktop/extensions-sdk/design/design-guidelines.md

```diff
@@ -2,6 +2,8 @@
 title: Design guidelines
 description: Docker extension design
 keywords: Docker, extensions, design
+redirect_from:
+- /desktop/extensions-sdk/design/design-guidelines/
```

Comment:
Is this the correct redirect? The URL seems to be the same URL as the document it's redirecting to (i.e., this markdown file)? `/desktop/extensions-sdk/design/design-guidelines.md`

Reply:
Looks like the broken URL ends with `-overview`; https://docs.docker.com/desktop/extensions-sdk/design/design-overview/

```suggestion
redirect_from:
- /desktop/extensions-sdk/design/design-overview/
```

Reply:
Thank you very much @thaJeztah I might have mixed things in my clipboard... 

I checked the history of the toc to see where https://docs.docker.com/desktop/extensions-sdk/design/overview/ used to point at. It seem there has been two reorg:
- https://github.com/docker/docs/pull/14697
- https://github.com/docker/docs/pull/14981

I moved back the overview on top and changed the name to be `index.md` as the root of the section. I think the content of it makes sense.